### PR TITLE
ref(crons): Use explicit grid for overviewTimeline

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: static/app/views/monitors/components/overviewTimeline/index.tsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ export function OverviewTimeline({monitorList}: Props) {
   );
 
   return (
-    <MonitorListPanel>
+    <MonitorListPanel rowCount={monitorList.length}>
```
Comment (davidenwang) [inline static/app/views/monitors/components/overviewTimeline/index.tsx:40]: if we call it rowCount should we just add the +1 in here? `rowCount={monitorList.length + 1}` or just change the name of the prop to `monitorCount`?

File: static/app/views/monitors/components/overviewTimeline/timelineScrubber.tsx [hunk 2]
```diff
@@ -65,26 +65,17 @@ export function GridLineOverlay({end, timeWindow}: Props) {
...
 const Overlay = styled('div')`
-  grid-row: 1;
   grid-column: 2;
-  height: 100%;
-  width: 100%;
-  position: absolute;
-`;
-
-const GridLineContainer = styled('div')`
+  grid-row: 1 / -1;
```
Comment (davidenwang) [inline static/app/views/monitors/components/overviewTimeline/timelineScrubber.tsx:77]: this is nice

**Review by davidenwang — APPROVED** [review]
amazing, one small comment but not blocking

File: static/app/views/monitors/components/overviewTimeline/index.tsx (hunk 1 above)
Comment (evanpurkhiser) [reply static/app/views/monitors/components/overviewTimeline/index.tsx:40]: I like it

**Pull request closed without merging**